jsonschema.hcl

```
@@ -0,0 +1,31 @@
+description = "jsonschema – The CLI for working with JSON Schema"
+homepage = "https://github.com/sourcemeta/jsonschema"
+binaries = ["bin/jsonschema"]
```

The build fail
```
fatal:hermit: jsonschema-9.6.1: no binaries found matching "/home/runner/.cache/hermit/pkg/jsonschema-9.6.1/bin/jsonschema" - this may indicate a corrupted or misconfigured package, try removing /home/runner/.cache/hermit/pkg/jsonschema-9.6.1 and hermit will reinstall it (may require sudo)

```

Should this be 
```suggestion
binaries = ["jsonschema"]
```